docs/gettsim_objects/means_testing.rst

```diff
@@ -0,0 +1,54 @@
+.. _means_testing:
+
+Assets for means testing
+========================
+
+| The table below gives an overview of all tangible and intangible assets which are deducted from the overall assets of a household when performing means testing for the respective transfer.
+
+| This documentation shall help to understand the composition of the :ref:`basic input variable <input_variables>` 'vermögen_bedürft_hh'.
+
+.. note::
+   | ALGII = Grundsicherung für Arbeitslose
+   | GSA/E = Grundsicherung im Alter/ bei Erwerbsminderung
+   | WoGe = Wohngeld
+   | KiZu = Kinderzuschlag
+
+   HS = hardship case
+
++-----------------------------------------------+---------+---------+---------+---------+
+| Not counted as assets                         |  ALGII  |  GSA/E  |  WoGe   |  KiZu   |
++===============================================+=========+=========+=========+=========+
+| Appropriate household goods                   |    x    |    x    |    x    |    x    |
++-----------------------------------------------+---------+---------+---------+---------+
+|| adequate motor vehicle for each              ||   x    || x (HC) ||   x    ||   x    |
+|| person capable of working                    ||        ||        ||        ||        |
++-----------------------------------------------+---------+---------+---------+---------+
+|| Assets for retirement provision and          ||   x    ||   x    ||   x    ||   x    |
+|| retirement assets                            ||        ||        ||        ||        |
++-----------------------------------------------+---------+---------+---------+---------+
+|| owner-occupied house/apartment of            ||   x    ||   x    ||   x    ||   x    |
```

Comment:
Make sure that the descriptions all start with a capital letter and don't have any capital letters thereafter.